src/test/kotlin/org/jitsi/nlj/test_utils/FakeClock.kt

```diff
@@ -0,0 +1,25 @@
+package org.jitsi.nlj.test_utils
```

Comment:
License headers?

Reply:
oops.  on a different computer to don't have my template i think.  will add.